src/protocols-table.js

```diff
@@ -42,7 +42,8 @@ Protocols.table = [
   [477, 0, 'ws'],
   [478, 0, 'wss'],
   [275, 0, 'libp2p-webrtc-star'],
-  [276, 0, 'libp2p-webrtc-direct']
+  [276, 0, 'libp2p-webrtc-direct'],
+  [277, 0, 'p2p-circuit']
```

Comment:
We haven't, so far, considered multiaddrs that are 'pseudo-encapsulate' other multiaddrs as if the other multiaddrs are part of size of the one that 'pseudo-encapsulates'. See the webrtc-star example or even ws.

I can understand the reasoning, it can be open for discussion, but the parsing is not made for that case right now.